bint.go

```diff
@@ -1,11 +1,18 @@
 package udecimal
 
 import (
+	"bytes"
 	"fmt"
 	"math/big"
 	"strings"
 )
 
+const (
+	// maxDigitU64 is the maximum digits of a number
+	// that can be safely stored in a uint64.
+	maxDigitU64 = 19
```

Comment:
You could add another comment for the 41 

Used in parseBint

Reply:
I think I already added comments for it, also it's not being reused like this const